src/renderer/App.tsx

```diff
@@ -127,6 +129,7 @@ class App extends Component<{}, State> {
         <D3Wrapper
           fileUploaded={this.state.fileUploaded}
           fileUpload={this.fileUpload}
+          serviceInfo={this.state.services}
```

Comment:
Same as previous comment: The props should have the same designation as the state.
`          services={this.state.services}`